content/en/docs/spin-operator/tutorials/pub-sub-with-redis.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Publish-Subscribe With Redis
+description: Learn how to create a Spin application that responds to messages on pub-sub Redis channels and runs in Kubernetes
+categories: [Spin Operator]
+tags: [Tutorials]
+weight: 100
+---
+
+## Prerequisites
+
+For this tutorial, we will be using:
+- [Spin](https://developer.fermyon.com/spin/v2/install) to build and deploy our event-driven WebAssembly application,
+- [Redis](https://redis.io/docs/install/install-redis/) to generate events in our real-time messaging scenario, and
+- [Rancher Desktop](https://rancherdesktop.io/) to manage Kubernetes on our Desktop. ([This page](../../spin-operator/tutorials/integrating-with-rancher-desktop.md) documents integrating Rancher Desktop and SpinKube.)
```

Comment:
I'm missing how we use Rancher in this tutorial.

Reply:
I'll leave the Rancher part out so as not to clutter the Redis message.